gvsbuild/projects/lz4.py

```diff
@@ -15,31 +15,25 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program; if not, see <http://www.gnu.org/licenses/>.
 
+from gvsbuild.utils.base_builders import Meson
 from gvsbuild.utils.base_expanders import Tarball
 from gvsbuild.utils.base_project import Project, project_add
 
 
 @project_add
-class Lz4(Tarball, Project):
+class Lz4(Tarball, Meson):
     def __init__(self):
         Project.__init__(
             self,
             "lz4",
             archive_url="https://github.com/lz4/lz4/archive/v1.9.3.tar.gz",
             archive_file_name="lz4-1.9.3.tar.gz",
             hash="030644df4611007ff7dc962d981f390361e6c97a34e5cbc393ddfbe019ffe2c1",
+            dependencies=["ninja", "meson"],
         )
+        self.add_param("-Ddefault_library=shared")
+        self.add_param("-Dbin_programs=true")
```

Comment:
Double check your options here

Reply:
Thanks. I was just copying the build options in the README. I submitted a PR to correct it: https://github.com/lz4/lz4/pull/1075